contents/docs/user-guides/groups.md

```diff
@@ -0,0 +1,293 @@
+
+You can't have something like: `transaction = [123456789, 12345678910]`. This is not possible.
+
+You can skip some groups if they don't make sense. For example, `transaction = 123456789`, `company = PostHog`, and no `pineapple_type` is valid as well.
+
+## Group Properties
+
+Every group can have properties associated with it. For example, if you have the `company` group, you could have properties defined on this group, like `company_name`, `revenue`, `user_count`, etc. etc.
+
+As another example, the `transactionID 12345679` group of `transaction` group type can have properties like `transaction_value`, `transaction_method`, etc. associated with them.
+
+These properties can be used in insights similar to how you use person properties.
+
+## How to think about Insights on groups
+
+Since there's no new events, you can use the same events you've been using for your insights. It's the same funnel you already have, aggregated differently.
+
+The groups determine what your insights aggregate on. For example, consider the funnel with two events: `User signed up -> User activated`
+
+Traditionally, you'd only have unique users in the funnel: Users from any and every company who're using your product and doing something important. Your conversion rate is per user. But what if you're interested in looking at this conversion rate across companies? Say, you're a B2B product and doing something important once per company is good enough. Then the funnel you want to track is `User signed up -> User activated` aggregated by unique companies, not users.
+
+That's a group aggregation over the company group type. It's the same events, aggregated differently.
+
+What if you wanted to breakdown this funnel by company name? Maybe there's specific companies that never converted, and you want to follow up with them to see what's going wrong. Like mentioned above, if you have group properties defined, you can breakdown by `company_name`. 
+
+## How to think about Feature Flags with groups
+
+Similar to insights, where you're aggregating events by group type, you can have Feature Flags that work on groups. This allows you to rollout a feature by company, instead of users. Preventing disruption when two users at the same company see a different experience.
+
+
+You can use feature flags as you normally would, except you need to select the group type you wish to "Match by", using the drop down by release conditions.
+
+You will also need to update your event tracking code (covered below), for the feature flag to be able to determine the groups of the current user.
+
+# How do I get started?
+
+## Getting access
+Groups is a paid feature, so you'll need to be on our [Scale or Enterprise plans](https://posthog.com/pricing) for self-hosted or have entered your credit-card on cloud.
+
+On-self hosted you must also be on the latest release: 1.31.0 or above.
```

Comment:
TODO: Link to license page (section from slack)

Reply:
# Feature availability
Group analytics is a premium feature that requires an additional subscription.
- If you're self-hosting PostHog, Group Analytics is available on the Scale & Enterprise plans. [Get a license now](https://license.posthog.com?utm_source=docs&utm_campaign=group-analytics-docs-license)
- If you're on PostHog Cloud, you just need to set up your billing information. Group Analytics is available on all our paid plans. [Set up billing](https://app.posthog.com/organization/billing?utm_source=docs&utm_campaign=group-analytics-docs-license)